oeps/oep-0037-bp-test-data.rst

```diff
@@ -94,60 +94,55 @@ The function can take one or more positional arguments if useful for using it in
 
 Data Files
 ----------
 
-When creating a more complex data set, especially one containing data to be loaded into multiple Open edX services, it is often best to describe the data in a YAML file.  The path or URL of this file can then be passed as an argument to each data loading function which uses the information in it to create database records for a particular service as shown above.  Requirements for such data sets vary widely, so we won't attempt to enforce a single standard schema for them, but each such file should declare the name of the schema it has chosen to use.  Such a file might look like this:
+Devstack dev data will be specified in a YAML file.  The path or URL of this file is passed to the ``load_dev_data`` management command, which uses the information in it to call the appropriate data generation function to create database records for a particular service as shown above. Such a file might look like this:
 
 .. code-block:: yaml
 
-    schema: dev_data_v1
-    users:
-        - admin
-        - staff
-        - student
-    enrollments:
-        - course: course-v1:edx+T101+2017_Q1
-          user: student
-        - course: course-v1:edx+MD101+2018_Q3
-          user: staff
+   users:
+    - username: verified
+      email: verified@example.com
+    - username: robot1
+      email: robot1@example.com
+  enrollments:
+    - username: verified
+      course_id: 'course-v1:edX+DemoX+Demo_Course'
+      mode: verified
+    - username: robot1
+      course_id: 'course-v1:edX+DemoX+Demo_Course'
+      mode: verified
     ...
 
-These data files should be as minimal as possible, containing just enough information for a data loading function familiar with this format to generate appropriate records using factory classes to fill in reasonable defaults for anything not explicitly specified.  This allows a single file to describe a data set which can be loaded into multiple services to allow testing cross-service functionality without binding the data set too closely to the current code or schema of any of those services.
+These data files should be as minimal as possible, containing just enough info for a data loading function familiar with this format to generate appropriate records using factory classes to fill in reasonable defaults for anything not explicitly specified.  This allows a single file to describe a data set which can be loaded into multiple services to allow testing cross-service functionality without binding the data set too closely to the current code or schema of any of those services.
```

Comment:
Why did we change `information` to `info`?

Reply:
I don't know. Hmm, will change it back.